src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +344,40 @@ Demos
 
 ## FAQ
 
+**Do I need the `--disable-gpu` flag?**
+
+Yes, for now.  The `--disable-gpu` flag is a temporary requirement to work around a few bugs. You won't need this
+flag in future versions of Chrome. See [https://crbug.com/546953#c152](https://bugs.chromium.org/p/chromium/issues/detail?id=546953#c152) and [https://crbug.com/695212](https://bugs.chromium.org/p/chromium/issues/detail?id=695212) for more information.
+
+**So I still need Xvfb?**
```

Comment:
That's not a detail of running headless though. I like to keep getting started experiences clean and minimal. Otherwise there's too much stuff to read before you get up and running.